src/woo_publications/publications/tests/test_archiving.py

```diff
@@ -0,0 +1,74 @@
+from django.test import TestCase
+
+from woo_publications.constants import ArchiveNominationChoices
+from woo_publications.metadata.models import InformationCategory
+from woo_publications.metadata.tests.factories import InformationCategoryFactory
+
+from ..archiving import get_retention_informatie_category
+
+
+class GetRetentionInformationCategoryTest(TestCase):
```

Comment:
I would rather see these cases only in the model tests and not any unit test for the `get_retention_informatie_category` helper. It is an implementation detail - as long as it's 100% covered via the model tests, we're good.

Now, if we change this function, move it, or delete it, we have to update these tests, even though functionally nothing changed. (Unit) tests must exist for the "public API", like the model methods, but these helpers that are only used to support the public API are not part of the contract to test and they're not intended to used anywhere else.